randovania/data/json_data/prime3.txt

```diff
@@ -69,6 +72,12 @@ Save Station B
 Asset id: 14403796150828511218
 > Dock to Repair Bay Shaft; Heals? False
   * Normal Door to Repair Bay Shaft/Dock to Save Station B
+  > Save Station
+      Trivial
+
+> Save Station; Heals? True
+  > Dock to Repair Bay Shaft
+      Trivial
 
 ----------------
 Flag Bridge
```

Comment:
Pretty much every room in Olympus is missing connections. While it can be skipped, it's worth adding to the database. Most connections are Trivial. Some might need events.